website/servers/server.php

```diff
@@ -82,13 +82,77 @@
 			die("invalid server location");
 		}
 
-		$PokemonServers[$id]['server'] = $server;
-		$PokemonServers[$id]['port'] = $port;
-	}
+		$oldServer = $PokemonServers[$id]['server'];
+		$oldPort = $PokemonServers[$id]['port'];
 
-	$owners = explode(',', $_POST['owner']);
-	foreach ($owners as $i=>$owner) $owners[$i] = $users->userid($owner);
-	$PokemonServers[$id]['owner'] = implode(',',$owners);
+		$cannotUpdate = false;
+		if (isset($serverinfo['bannedhosts']) && trim($server) !== trim($oldServer)) {
+			foreach ($serverinfo['bannedhosts'] as $bannedhost) {
+				$bannedhost = trim($bannedhost);
+				if ($bannedhost && strpos($server, $bannedhost) > -1) {
+					$cannotUpdate = true;
+					echo(
+						"<br /><strong style=\"color:red\">You cannot use the host " . htmlspecialchars ($server) .
+						" as it is from a blacklisted host.</strong><br />"
+					);
+					break;
+				}
+			}
+		}
+
+		foreach ($PokemonServers as $curserver) {
+			if ($curserver['server'] === $server) {
+				$cannotUpdate = true;
+				echo(
+					"<br /><strong style=\"color:red\">You cannot use the host " . $server .
+					" as it is already in use by another server.</strong><br />"
+				);
```

Comment:
```suggestion
				echo "<br /><strong style=\"color:red\">You can't use the host " . htmlspecialchars($server) . " because it is already in use by another server.</strong><br />";
```